sopel/modules/pronouns.py

```diff
@@ -78,30 +186,30 @@ def set_pronouns(bot, trigger):
         return
 
     disambig = ''
-    requested_pronoun_split = pronouns.split("/")
-    if len(requested_pronoun_split) < 5:
+    split_request = pronouns.split("/")
+    if len(split_request) < 5:
         matching = []
-        for known_pronoun_set in KNOWN_SETS.values():
-            known_pronoun_split = known_pronoun_set.split("/")
-            if known_pronoun_set.startswith(pronouns + "/") or (
-                len(requested_pronoun_split) == 3
+        for known_set in bot.memory['pronoun_sets'].values():
+            split_set = known_set.split("/")
+            if known_set.startswith(pronouns + "/") or (
+                len(split_request) == 3
```

Comment:
I don't really understand the change in names here. Sure, I don't really like "known_pronoun_set", I'd just use "known_pronouns".

Reply:
And yeah, both "split_request" and "request_pronoun_split" tries to encode too much information in one name. I'd just go with "requested_pronoun". 🤷